splash/qtrender_lua.py

```diff
@@ -415,16 +416,39 @@ def error(error_info):
         return cmd
 
     @command(async=True)
-    def go(self, url, baseurl=None, headers=None):
+    def go(self, url, baseurl=None, headers=None, http_method="GET", body=None, formdata=None):
+
         if url is None:
             raise ScriptError({
                 "argument": "url",
                 "message": "'url' is required for splash:go",
             })
 
+        http_method = http_method.upper()
+        if http_method not in ["POST", "GET"]:
+            raise ScriptError("Unsupported HTTP method {}".format(http_method))
```

Comment:
Sorry, I changed the way errors are reported in master. The way to go now is to instantiate ScriptError with a dict (check other methods for examples). Could you please update this?